run_dir/static/js/project_samples.js

```diff
@@ -499,13 +499,20 @@ function load_all_udfs(){
 
       // Create the links for review and display the banner
       else if (prettify(key) == 'pending_reviews'){
-          review_links=''
-          $.each(value,function(index,limsid){
-              review_links+="<a href='https://genologics.scilifelab.se:8443/clarity/work-complete/"+limsid+"'>Go to the Lims Step</a> ";
+        review_links = '';
+        if($.isArray(value)){
+          $.each(value, function(index, limsid){
+              review_links += "<a href='https://genologics.scilifelab.se:8443/clarity/work-complete/"+limsid+"'>Go to the Lims Step</a> ";
```

Comment:
As @ewels said about some other PR, having an explicit URL in code is not so good. Ought to be moved out to some config at some point.